RELEASE.md

```diff
@@ -1,78 +1,164 @@
-1. [NGINX](https://github.com/kubernetes/ingress-nginx/tree/main/images/nginx)
+## 1. [NGINX Base Image](https://github.com/kubernetes/ingress-nginx/tree/main/images/nginx)
 
-* Open pull request
+A. Open pull request
   
-If you are updating any component in [build.sh](images/nginx/rootfs/build.sh) please also update the SHA256 checksum of that component as 
-  well, the cloud build will fail with an exit 10 if not.  
+  If you are updating any component in [build.sh](images/nginx/rootfs/build.sh) please also update the SHA256 checksum of that component as well, the cloud build will fail with an exit 10 if not.  
 Example [NGINX_VERSION](images/nginx/rootfs/build.sh#L21),
 [SHA256](images/nginx/rootfs/build.sh#L124) 
-* Merge
-* Wait for [cloud build](https://console.cloud.google.com/cloud-build/builds?project=k8s-staging-ingress-nginx)
 
-1a. Promote images:
+B. Merge
 
-Open pull request to promote staging image:
-[add sha - version](https://github.com/kubernetes/k8s.io/blob/main/k8s.gcr.io/images/k8s-staging-ingress-nginx/images.yaml#L1)
```

Comment:
reference to line 27 makes more sense IMHO

Reply:
Line27 is empty in my clone. What did I miss plz ?

Reply:
Line 27 or 28 now refers explicitly to the block where the new entry has to be made